services/meta/ota.md

```diff
@@ -1,88 +1,114 @@
 
 * `ota_end` is an object with the following structure:
 
-| Field          | Type   | Example          | Description                                                           |
-|----------------|--------|------------------|-----------------------------------------------------------------------|
-| success        | bool   | `true`           | Shows whether the update was successful or not.                       |
-| error          | string | `"invalid_file"` | In case of a failed update describes fail reason, otherwise empty.    |
+| Field   | Type   | Example           | Description                                                                                                                          |
+|---------|--------|-------------------|--------------------------------------------------------------------------------------------------------------------------------------|
+| success | bool   | `false`           | Shows whether the update was successful or not.                                                                                      |
+| error   | string | `"invalid_image"` | If success is false it states the reason for update failure, see [`error`](#definitions) definition for list of all possible values. |
+
+* `error` represents a cause of the failed update and is one of the following: `low_battery`, `invalid_image`, `not_upgradable`, `needs_user_action`, `other`.
 
 ## Examples
 
+* Example of a report sent when the update process starts:
+
 ```json
 {
-   "type": "evt.ota_progress.report",
-   "serv": "ota",
-   "val_t": "int_map",
-   "val": {
-      "progress": 10,
-      "remaining_min": 40,
-      "remaining_sec": 3
-   }
+  "type": "evt.ota_start.report",
+  "serv": "ota",
+  "val_t": "null",
+  "val": null,
+  "props": {},
+  "tags": [],
+  "src": "-",
+  "ver": "1",
+  "uid": "e604e951-7afb-4f96-981b-62e905757686",
+  "topic": "pt:j1/mt:evt/rt:dev/rn:zigbee/ad:1/sv:ota/ad:1_2"
 }
 ```
 
+* Example of message sent during the transfer of the image phase of the OTA update process:
+
 ```json
 {
-   "type": "evt.ota_end.report",
-   "serv": "ota",
-   "val_t": "object",
-   "val": {
-      "success": true,
-      "error": ""
-   }
+  "type": "evt.ota_progress.report",
+  "serv": "ota",
+  "val_t": "int_map",
+  "val": {
+    "progress": 10,
+    "remaining_min": 40,
+    "remaining_sec": 3
+  },
+  "props": {},
+  "tags": [],
+  "src": "-",
+  "ver": "1",
+  "uid": "e604e951-7afb-4f96-981b-62e905757686",
+  "topic": "pt:j1/mt:evt/rt:dev/rn:zigbee/ad:1/sv:ota/ad:1_2"
 }
 ```
 
+* Example of a report sent when the update process is completed successfully:
+
 ```json
 {
-   "type": "evt.ota_status.report",
-   "serv": "ota",
-   "val_t": "object",
-   "val": {
-      "state": "IN PROGRESS",
-      "substate": "TRANSFERING",
-      "last_update": 12625242636
-   }
+  "type": "evt.ota_end.report",
+  "serv": "ota",
+  "val_t": "object",
+  "val": {
+    "success": true
+  },
```

Comment:
I assume we leave it like that, so it could possibly be extendable in future right?